vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go

```
@@ -338,6 +349,29 @@ func (m *StorenodeCycle) GetActiveStorenode() peer.ID {
 	return m.activeStorenode
 }
 
+func (m *StorenodeCycle) GetActiveStorenodePeerInfo() peer.AddrInfo {
+	m.RLock()
+	defer m.RUnlock()
+
+	if m.storenodeConfigProvider == nil {
+		m.logger.Debug("storenodeConfigProvider not yet setup")
+		return peer.AddrInfo{}
```

We are hiding the error and I'm wondering if this is the best approach :) Applies elsewhere